app/decorators/plant_trial_submission_decorator.rb

```diff
@@ -43,7 +43,7 @@ def plant_population
 
   def trait_descriptors
     tds = object.content.step02.trait_descriptor_list.try(:select, &:present?) || []
-    numeric_tds = tds.select { |td| td.to_s.match /\A\d+\z/ }
+    numeric_tds = tds.select { |td| td.to_s.match(/\A\d+\z/) }
```

Comment:
I used `existing_` before as it seemed more appropriate than `old_` but I can change it to `old_` for the sake of consistency.

It seems to me (from checking the code) that naming a TD with a number will result in:

- either ignoring that TD altogether (if no TD with given id exists),
- or assigning existing TD instead of using TD data provided.
  
Either way, it is a bug.